packages/connectors-web/src/utils/resolveConnector.ts

```diff
@@ -0,0 +1,30 @@
+import type { Config, Connector } from "wagmi";
+import type { AuthClient, Signer } from "@alchemy/signer";
+
+/**
+ * Extended connector interface that includes Alchemy Auth-specific methods
+ */
+export interface AlchemyAuthConnector extends Connector {
+  getAuthClient(): Promise<AuthClient>;
+  getSigner(): Promise<Signer>;
+  setSigner(signer: Signer): void;
+}
+
+/**
+ * Helper function to locate and return the Alchemy Auth connector from the config
+ *
+ * @param {Config} config - The Wagmi config containing connectors
+ * @param {string} [id] - The connector ID to search for (defaults to "alchemyAuth")
+ * @returns {AlchemyAuthConnector} The Alchemy Auth connector
+ * @throws {Error} If the connector is not found
+ */
+export function resolveAlchemyAuthConnector(
```

Comment:
not sure if we need this helper method, but it was confusing to figure out how to get the alchemy Auth connector when implementing the OTP actions